packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -636,10 +658,22 @@ class S3Source extends Source {
   }
 }
 
+/**
+ * wip
+ */
+export interface GithubContextProps extends ThirdPartyGitSourceProps {
+  /**
+   * This parameter is used for the `context` parameter in the GitHub commit status.
+   *
+   * @example 'My build #$CODEBUILD_BUILD_NUMBER'
+   */
+    readonly buildStatusContext: string
```

Comment:
This time, the property is misaligned:

```suggestion
  /**
   * This parameter is used for the `context` parameter in the GitHub commit status.
   *
   * @example 'My build #$CODEBUILD_BUILD_NUMBER'
   */
  readonly buildStatusContext: string;
```